static/manager-dashboard/src/shared/VerticalBarChart.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+import { Bar } from 'react-chartjs-2';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend
+);
+
+const VerticalBarChart = (props) => {
+    //props need: text, xAxisLabels, dataset1Label, dataSet1
+    const options = {
+        responsive: true,
+        plugins: {
+            legend: {
+            position: 'top',
+            },
+            title: {
+            display: true,
+            text: props.text,
+            },
+        },
+    };
+
+    const labels = props.xAxisLabels;
+
+    const data = {
+        labels,
+        datasets: [
```

Comment:
If you're allowing multiple datasets, reckon pass the entire dataset as a prop. This locks people in, forcing them to pass in 2 datasets right?

Reply:
Yeah was previously restricted to 2 data sets. Have now updated it to pass in the whole data set as a prop, so there's no longer the restriction in place.